infrastructure/tooling/src/utils/command.js

```diff
@@ -58,7 +58,9 @@ exports.execCommand = async ({
       if (code === 0 || ignoreReturn) {
         resolve(output);
       } else {
-        reject(new Error(`Nonzero exit status ${code}; see ${logfile} for details`));
+        utils.status({message: output});
```

Comment:
Stray status or do we always want to show it?

Reply:
Yeah, I think that doesn't work because the status is hidden once the task fails.  I'll remove.